packages/database/src/TupaiaDatabase.js

```diff
@@ -513,6 +513,10 @@ function buildQuery(connection, queryConfig, where = {}, options = {}) {
     query.returning('*');
   }
 
+  if (process.env.DB_VERBOSE === 'true') {
+    winston.info(query.toString());
+  }
```

Comment:
This is unrelated but helpful. Allows running e.g. `DB_VERBOSE=true yarn workspace @tupaia/meditrak-server start-dev` to see all queries logged to the console. To see the same in tests, you do also need to update the winston transports in `scaffolding.test.js` like so:
![image](https://user-images.githubusercontent.com/1274422/134827308-52bcf020-1225-4f36-9747-106fadf3ef90.png)

Reply:
Nice one! The other day I also realised we should not silence error logging in tests by default. Some times they indicate an issue in the tested code (which is not picked by an assertion).

We can then silence errors selectively (ideally silence specific types of errors, or test the code in a level where errors are thrown rather than logged)

Reply:
True! Left the tests log handling as out of scope here, but if you felt like doing that go for it. We'd have to be fairly assiduous in silencing expected logs so that logs became more meaningful - at the moment I feel I see logs in the tests and just ignore them because most of the time they're just a side effect

Reply:
Started doing that a few weeks ago, took more time than would be justified at that stage, and left it in the middle. Ideally we'd catch specific error types, and log everything else